重命名下载标签时不显示原标签

### 复现步骤 / Step to reproduce

0.更新应用到最新版本
1.打开自己的下载
2.点击下载标签图案
3.重命名任意一个下载标签
4.所有的下载标签在重命名的时候都是空的，无法通过任何直接手段读取或者复制原标签的内容
5.（新建的下载标签也同样无法读取其中内容）新建一个下载标签，命名为1，在重命名的时候，仍然无法读取到1，输入2则下载标签直接被修改为2

### 预期行为 / Expected behaviour

在旧版本里，他应该是可以读取标签的信息的
也就是说，原本标签内容为：触手
我在重命名标签时，重命名的栏目里仍然应该存在“触手”这2个字，这样我就可以直接对其进行更改，例如改为：触手+巨乳
但是由于现在的读取bug导致无法完成此类操作

### 实际行为 / Actual Behaviour

实际上他现在在重命名下载标签时，重命名的栏目里永远是空的，无论是旧版本时就建立的标签无法沿着之前的名字重命名，新建的也仍然只能覆盖式的重命名，且无法读取标签名字

### 应用日志 / App logs

======== PackageInfo ========
PackageName=moe.tarsin.ehviewer
VersionName=1.10.1-gms
VersionCode=180048
CommitSha=7008209
BuildTime=2023/12/19 10:38

======== DeviceInfo ========
BOARD=NTH
BOOTLOADER=unknown
BRAND=HONOR
CPU_ABI=arm64-v8a
CPU_ABI2=
DEVICE=HNNTH
DISPLAY=NTH-AN00 4.2.0.199(C00E198R2P9)
FINGERPRINT=HONOR/NTH-AN00/HNNTH:11/HONORNTH-AN00/4.2.0.199C00:user/release-keys
FINGERPRINTEX=HONOR/NTH-AN00/HNNTH:11/HONORNTH-AN00/4.2.0.199C00:user/release-keys
HARDWARE=qcom
HOST=cn-snxy1c-7f01ed4931643453248183-5b7d65f6d9-7g59b
HWFINGERPRINT=///NTH-LGRP1-CHN 4.2.0.199/NTH-AN00-CUST 4.2.0.198(C00)/NTH-AN00-PRELOAD 4.2.0.9(C00R2)//patch01
ID=HONORNTH-AN00
IS_DEBUGGABLE=false
IS_EMULATOR=false
MANUFACTURER=HONOR
MODEL=NTH-AN00
PERMISSIONS_REVIEW_REQUIRED=true
PRODUCT=NTH-AN00
RADIO=unknown
SERIAL=unknown
SUPPORTED_32_BIT_ABIS=armeabi-v7a, armeabi
SUPPORTED_64_BIT_ABIS=arm64-v8a
SUPPORTED_ABIS=arm64-v8a, armeabi-v7a, armeabi
TAGS=release-keys
TIME=1643454156000
TYPE=user
UNKNOWN=unknown
USER=test
ACTIVE_CODENAMES=
BASE_OS=
CODENAME=REL
INCREMENTAL=4.2.0.199C00
PREVIEW_SDK_FINGERPRINT=REL
PREVIEW_SDK_INT=0
RELEASE=11
RELEASE_OR_CODENAME=11
SDK=30
SDK_INT=30
SECURITY_PATCH=2022-01-01
MEMORY=18.8 MiB
MEMORY_NATIVE=23.3 MiB
MEMORY_MAX=384.0 MiB
MEMORY_TOTAL=7.2 GiB

--------- beginning of crash
11-26 05:12:22.868 22594 29072 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-7
11-26 05:12:22.868 22594 29072 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 22594
11-26 05:12:22.868 22594 29072 E AndroidRuntime: java.lang.NullPointerException
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at GB.r(Unknown Source:201)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at nB.s(Unknown Source:11)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at cA.run(Unknown Source:112)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at N1.run(Unknown Source:122)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	at nu.run(Unknown Source:91)
11-26 05:12:22.868 22594 29072 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@a40e382, Dispatchers.IO]
11-27 03:09:51.158 25842  8890 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-9
11-27 03:09:51.158 25842  8890 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 25842
11-27 03:09:51.158 25842  8890 E AndroidRuntime: java.lang.NullPointerException
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at GB.r(Unknown Source:201)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at nB.s(Unknown Source:11)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at cA.run(Unknown Source:112)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at N1.run(Unknown Source:122)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	at nu.run(Unknown Source:91)
11-27 03:09:51.158 25842  8890 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@54ff33e, Dispatchers.IO]
11-30 14:11:19.054  5792 18392 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-10
11-30 14:11:19.054  5792 18392 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 5792
11-30 14:11:19.054  5792 18392 E AndroidRuntime: java.lang.NullPointerException
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at GB.r(Unknown Source:201)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at nB.s(Unknown Source:11)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at cA.run(Unknown Source:112)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at N1.run(Unknown Source:122)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	at nu.run(Unknown Source:91)
11-30 14:11:19.054  5792 18392 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@a09cbc1, Dispatchers.IO]
12-06 14:35:15.745  7432 10329 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-9
12-06 14:35:15.745  7432 10329 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 7432
12-06 14:35:15.745  7432 10329 E AndroidRuntime: java.lang.NullPointerException
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-06 14:35:15.745  7432 10329 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@f5d660, Dispatchers.IO]
12-09 03:58:37.496 13295  8302 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-3
12-09 03:58:37.496 13295  8302 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 13295
12-09 03:58:37.496 13295  8302 E AndroidRuntime: java.lang.NullPointerException
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-09 03:58:37.496 13295  8302 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@e9d14dd, Dispatchers.IO]
12-11 02:12:16.951 20760 20760 E AndroidRuntime: FATAL EXCEPTION: main
12-11 02:12:16.951 20760 20760 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 20760
12-11 02:12:16.951 20760 20760 E AndroidRuntime: java.lang.ExceptionInInitializerError
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at jF.onApplyThemeResource(Unknown Source:9)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.view.ContextThemeWrapper.initializeTheme(ContextThemeWrapper.java:216)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.view.ContextThemeWrapper.setTheme(ContextThemeWrapper.java:147)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.Activity.setTheme(Activity.java:5257)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at r9.setTheme(Unknown Source:0)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4048)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4295)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:93)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2532)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:228)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: Caused by: java.lang.IllegalStateException: This should NEVER be called on the main thread! Current: Thread[main,5,main]
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	at SJ0.<clinit>(Unknown Source:942)
12-11 02:12:16.951 20760 20760 E AndroidRuntime: 	... 17 more
12-12 13:58:30.199 28408 31508 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-1
12-12 13:58:30.199 28408 31508 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 28408
12-12 13:58:30.199 28408 31508 E AndroidRuntime: java.lang.NullPointerException
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-12 13:58:30.199 28408 31508 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@9608dce, Dispatchers.IO]
12-13 15:09:35.733 17033 17065 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-3
12-13 15:09:35.733 17033 17065 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 17033
12-13 15:09:35.733 17033 17065 E AndroidRuntime: java.lang.NullPointerException
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-13 15:09:35.733 17033 17065 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@6973a33, Dispatchers.IO]
12-15 19:31:52.925  6964  6964 E AndroidRuntime: FATAL EXCEPTION: main
12-15 19:31:52.925  6964  6964 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 6964
12-15 19:31:52.925  6964  6964 E AndroidRuntime: java.lang.ExceptionInInitializerError
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at jF.onApplyThemeResource(Unknown Source:9)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.view.ContextThemeWrapper.initializeTheme(ContextThemeWrapper.java:216)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.view.ContextThemeWrapper.setTheme(ContextThemeWrapper.java:147)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.Activity.setTheme(Activity.java:5257)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at r9.setTheme(Unknown Source:0)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4048)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4295)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:93)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2532)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:228)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: Caused by: java.lang.IllegalStateException: This should NEVER be called on the main thread! Current: Thread[main,5,main]
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	at SJ0.<clinit>(Unknown Source:942)
12-15 19:31:52.925  6964  6964 E AndroidRuntime: 	... 17 more
12-16 04:23:04.669  2617 27070 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-18
12-16 04:23:04.669  2617 27070 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 2617
12-16 04:23:04.669  2617 27070 E AndroidRuntime: java.lang.NullPointerException
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-16 04:23:04.669  2617 27070 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@3cb11f1, Dispatchers.IO]
12-17 06:51:00.089 24519  6774 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-19
12-17 06:51:00.089 24519  6774 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 24519
12-17 06:51:00.089 24519  6774 E AndroidRuntime: java.lang.NullPointerException
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at GB.r(Unknown Source:201)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at nB.s(Unknown Source:11)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at Fd.r(Unknown Source:8)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at cA.run(Unknown Source:112)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at N1.run(Unknown Source:122)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at sV0.run(Unknown Source:2)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	at nu.run(Unknown Source:91)
12-17 06:51:00.089 24519  6774 E AndroidRuntime: 	Suppressed: zy: [rQ0{Cancelling}@a45476c, Dispatchers.IO]
12-17 21:21:13.296   985   985 E AndroidRuntime: FATAL EXCEPTION: main
12-17 21:21:13.296   985   985 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 985
12-17 21:21:13.296   985   985 E AndroidRuntime: java.lang.ClassCastException: Dc0 cannot be cast to aV
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at F81.n(Unknown Source:453)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at zW.a(Unknown Source:4)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at W01.onSingleTapUp(Unknown Source:10)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at gX.onSingleTapUp(Unknown Source:8)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.GestureDetector.onTouchEvent(GestureDetector.java:798)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Hz.a(Unknown Source:27)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.E(Unknown Source:20)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onInterceptTouchEvent(Unknown Source:9)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.checkInterception(ViewGroup.java:2784)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2830)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3314)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Gu0.n(Unknown Source:56)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Af.n(Unknown Source:1646)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at iy0.E1(Unknown Source:29)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Eu0.a(Unknown Source:85)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at rd.p0(Unknown Source:69)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Tl0.e(Unknown Source:53)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at Tl0.e(Unknown Source:140)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at em0.e(Unknown Source:15)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at jy.c(Unknown Source:32)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at TQ0.k(Unknown Source:156)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.H(Unknown Source:78)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.p(Unknown Source:213)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(Unknown Source:75)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:698)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2030)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4300)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at J9.dispatchTouchEvent(Unknown Source:2)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:644)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14757)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7265)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7060)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6467)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6660)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6717)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9533)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9435)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9388)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9730)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:271)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:362)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:185)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-17 21:21:13.296   985   985 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: FATAL EXCEPTION: main
12-17 21:21:32.967  3066  3066 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 3066
12-17 21:21:32.967  3066  3066 E AndroidRuntime: java.lang.ClassCastException: Dc0 cannot be cast to aV
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at F81.n(Unknown Source:453)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at zW.a(Unknown Source:4)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at W01.onSingleTapUp(Unknown Source:10)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at gX.onSingleTapUp(Unknown Source:8)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.GestureDetector.onTouchEvent(GestureDetector.java:798)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Hz.a(Unknown Source:27)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.E(Unknown Source:20)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onInterceptTouchEvent(Unknown Source:9)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.checkInterception(ViewGroup.java:2784)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2830)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3314)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Gu0.n(Unknown Source:56)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Af.n(Unknown Source:1646)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at iy0.E1(Unknown Source:29)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Eu0.a(Unknown Source:85)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at rd.p0(Unknown Source:69)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Tl0.e(Unknown Source:53)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at Tl0.e(Unknown Source:140)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at em0.e(Unknown Source:15)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at jy.c(Unknown Source:32)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at TQ0.k(Unknown Source:156)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.H(Unknown Source:78)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.p(Unknown Source:213)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(Unknown Source:75)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:698)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2030)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4300)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at J9.dispatchTouchEvent(Unknown Source:2)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:644)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14757)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7265)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7060)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6467)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6660)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:21:32.967  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6717)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9533)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9435)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9388)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9730)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:271)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:362)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:185)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-17 21:21:32.968  3066  3066 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: FATAL EXCEPTION: main
12-17 21:22:08.260  3645  3645 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 3645
12-17 21:22:08.260  3645  3645 E AndroidRuntime: java.lang.ClassCastException: Dc0 cannot be cast to aV
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at F81.n(Unknown Source:453)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at zW.a(Unknown Source:4)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at W01.onSingleTapUp(Unknown Source:10)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at gX.onSingleTapUp(Unknown Source:8)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.GestureDetector.onTouchEvent(GestureDetector.java:798)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Hz.a(Unknown Source:27)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.E(Unknown Source:20)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onInterceptTouchEvent(Unknown Source:9)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.checkInterception(ViewGroup.java:2784)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2830)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3314)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Gu0.n(Unknown Source:56)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Af.n(Unknown Source:1646)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at iy0.E1(Unknown Source:29)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Eu0.a(Unknown Source:85)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at rd.p0(Unknown Source:69)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Tl0.e(Unknown Source:53)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at Tl0.e(Unknown Source:140)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at em0.e(Unknown Source:15)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at jy.c(Unknown Source:32)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at TQ0.k(Unknown Source:156)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.H(Unknown Source:78)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.p(Unknown Source:213)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(Unknown Source:75)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:698)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2030)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4300)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at J9.dispatchTouchEvent(Unknown Source:2)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:644)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14757)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7265)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7060)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6467)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6660)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6717)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9533)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9435)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9388)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9730)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:271)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:362)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:185)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-17 21:22:08.260  3645  3645 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: FATAL EXCEPTION: main
12-17 21:22:28.554  4671  4671 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 4671
12-17 21:22:28.554  4671  4671 E AndroidRuntime: java.lang.ClassCastException: Dc0 cannot be cast to aV
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at F81.n(Unknown Source:453)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at zW.a(Unknown Source:4)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at W01.onSingleTapUp(Unknown Source:10)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at gX.onSingleTapUp(Unknown Source:8)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.GestureDetector.onTouchEvent(GestureDetector.java:798)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Hz.a(Unknown Source:27)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.E(Unknown Source:20)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onInterceptTouchEvent(Unknown Source:9)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.checkInterception(ViewGroup.java:2784)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2830)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3314)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Gu0.n(Unknown Source:56)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Af.n(Unknown Source:1646)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at iy0.E1(Unknown Source:29)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Eu0.a(Unknown Source:85)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at rd.p0(Unknown Source:69)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Tl0.e(Unknown Source:53)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at Tl0.e(Unknown Source:140)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at em0.e(Unknown Source:15)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at jy.c(Unknown Source:32)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at TQ0.k(Unknown Source:156)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.H(Unknown Source:78)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.p(Unknown Source:213)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(Unknown Source:75)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:698)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2030)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.app.Activity.dispatchTouchEvent(Activity.java:4300)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at J9.dispatchTouchEvent(Unknown Source:2)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:644)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14757)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7265)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7060)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6467)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6660)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6717)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9533)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9435)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9388)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9730)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:271)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:362)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:185)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
12-17 21:22:28.554  4671  4671 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-8
12-19 01:49:23.159  4828  5178 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 4828
12-19 01:49:23.159  4828  5178 E AndroidRuntime: java.lang.NullPointerException
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at RH.s(Unknown Source:198)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at AH.w(Unknown Source:11)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at He.j(Unknown Source:8)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at RF.run(Unknown Source:112)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at g2.run(Unknown Source:123)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at Qa1.run(Unknown Source:2)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	at Fx.run(Unknown Source:95)
12-19 01:49:23.159  4828  5178 E AndroidRuntime: 	Suppressed: oE: [Z51{Cancelling}@250d8c1, Dispatchers.IO]
--------- beginning of main
12-22 02:54:56.397 30783 30783 E libc    : Access denied finding property "persist.device_config.runtime_native.use_app_image_startup_cache"
12-22 02:54:56.398 30783 30783 E tarsin.ehviewe: LoadAppImageStartupCache enabled : 1
12-22 02:54:56.411 30783 30783 E tarsin.ehviewe: Not starting debugger since process cannot load the jdwp agent.
12-22 02:54:56.449 30783 30806 E BehaviorCollectManager: Fail to acquire dataAnalyzerService...


### 备注 / Addition details

可以的话，能还原旧版本的移除标签时弹窗来确认是否移除标签吗？有的下载标签里面有几十个本子，有时候手残不小心移除了会十分的麻烦。我记得旧版本是有这个功能的，但是新版本貌似取消了？

### 浏览站点 / Browsing site

ExHentai

### EhViewer 版本号 / EhViewer version code

1.10.1

### EhViewer CI 版本 / EhViewer CI Version

7008209

### Android 系统版本 / Android version

Magic UI 4.2 ; Android 11

### 设备型号 / Device model

Honor 50

### SoC 型号 / Soc model

骁龙778G

### 自查步骤 / Verify steps

- [] 如果您有足够的时间和能力，并愿意为此提交 PR ，请勾上此复选框 / Pull request is welcome. Check this if you want to start a pull request
- [X] 您已搜索过 [Issue Tracker](https://github.com/FooIbar/EhViewer/issues)，没有找到类似内容 / I have searched on [Issue Tracker](https://github.com/FooIbar/EhViewer/issues), No duplicate or related open issue has been found
- [X] 您确保这个 Issue 只提及一个问题。如果您有多个问题报告，烦请发起多个 Issue / Ensure there is only one bug report in this issue. Please make mutiply issue for mutiply bugs
- [X] 您确保已使用最新 [CI 版本](https://github.com/FooIbar/EhViewer/actions)测试，并且该问题在最新 CI 版本中并未解决 / This bug have not solved in latest [CI version](https://github.com/FooIbar/EhViewer/actions)

--- a/app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt
+++ b/app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt
@@ -277,7 +277,7 @@ fun DownloadsScreen(navigator: DestinationsNavigator) {
                                         IconButton(
                                             onClick = {
                                                 coroutineScope.launch {
-                                                    val new = dialogState.awaitInputText(title = renameLabel, hint = labelsStr) { text ->
+                                                    val new = dialogState.awaitInputText(initial = item, title = renameLabel, hint = labelsStr) { text ->
                                                         when {
                                                             text.isBlank() -> labelEmpty
                                                             text == defaultName -> defaultInvalid

